bacon/commons/Constants.swift

```diff
@@ -7,25 +7,46 @@
 //
 
 import Foundation
+import Contacts
+import UIKit
 
 class Constants {
     // Global
     static let currencySymbol = Locale.current.currencySymbol ?? "$"
+    static let defaultPostalAddress = CNPostalAddress()
+    static let warningTitle = "🐷WARNING🐷"
+    static let coreFailureMessage = "Unable to run Bacon!"
+    static let transactionAddFailureMessage = "Unable to add transaction!"
+    static let transactionLoadFailureMessage = "Unable to load transactions!"
 
-    // Add Transaction Page Constants
+    // Add Transaction Page
     static let defaultTransactionType = TransactionType.expenditure
     static let defaultCategory = TransactionCategory.food
     static let defaultCategoryString = Constants.defaultCategory.rawValue
     static let defaultAmount: Decimal = 0
     static let defaultAmountString = "0"
     static let defaultDescription = ""
 
+    // Trsansactions Page
+    static let defaultImage = UIImage(named: "dummy")
+    static let animatoinDuration = [0.26, 0.20, 0.20]
+
     // Date Formatter
     static func getDateFormatter() -> DateFormatter {
         let formatter = DateFormatter()
         formatter.dateFormat = "yyyy-MM-dd HH:mm:ss"
         return formatter
     }
+    static func getDateOnlyFormatter() -> DateFormatter {
```

Comment:
Should all these formatter methods accept a dateFormat string argument, so you don't have to hardcode it in?

Reply:
This one also similar argument as above, like I would want to standardise the only ways we can format the date. (say you can't pass in "yyyy-MM")